tests/mocks/control.py

```
@@ -57,38 +57,38 @@ def _handle_request_payload(data: dict) -> dict:
     headers = list(
         [(k, v) for (k, v) in data.get("headers", []) if not k.startswith("x-alltrue-")]
     )
     response = _handle_request_payload(data)
-    response["processed_headers"] = headers
+    response["headers"] = headers
     return response
 
 
-@app.post("/v1/llm-firewall/chat/batch/process-input/{proxy_type}")
+@app.post("/v1/rules/batch/process-input/{proxy_type}")
 async def chat_batch_request(request: Request, proxy_type: str):
```

Ruff (0.12.2)

Unused function argument: `proxy_type`

(ARG001)